### DISCUSSION TITLE
file gone

### DISCUSSION THREAD

**Discussion by rlarudqls630** [photoprism/photoprism#4103] [General]
I tried to see my photos after I updated photoprism in TrueNAS.
However, I am only left with yml files and black phtos in web ui.

Can any one help me get back my photos?
![Screenshot 2024-03-14 at 15 09 18](https://github.com/photoprism/photoprism/assets/11022460/aa6fe70a-3eb7-4ebf-9c12-d075df4c94a8)
![Screenshot 2024-03-14 at 15 11 18](https://github.com/photoprism/photoprism/assets/11022460/a5ab1d98-f3ba-469e-8ce7-a5e270aaa063)

**Comment by graciousgrey** [comment]
Please see out troubleshooting checklist for this case: https://docs.photoprism.app/getting-started/troubleshooting/#broken-thumbnails
